staging/src/k8s.io/apiserver/pkg/util/flowcontrol/default-objects.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	rmv1a1 "k8s.io/api/flowcontrol/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/authentication/user"
+)
+
+// immutable default priority-levels
+var (
+	DefaultPriorityLevelConfigurationExempt = pl(
+		rmv1a1.PriorityLevelConfigurationNameExempt,
+		rmv1a1.PriorityLevelConfigurationSpec{Exempt: true},
+	)
+	DefaultPriorityLevelConfigurationDefault = pl(
+		rmv1a1.PriorityLevelConfigurationNameDefault,
+		rmv1a1.PriorityLevelConfigurationSpec{
+			GlobalDefault:            true,
+			AssuredConcurrencyShares: 100,
+			Queues:                   128,
+			HandSize:                 6,
+			QueueLengthLimit:         100,
+		})
+)
+
+// immutable default flow-schemas
+var (
+	DefaultFlowSchemaSystemTop = fs(
+		"system-top",
+		rmv1a1.PriorityLevelConfigurationNameDefault,
+		1500, "",
+		rmv1a1.PolicyRuleWithSubjects{
+			Subjects: append(
+				users(user.APIServerUser),
+				groups(user.SystemPrivilegedGroup)...,
+			),
+			Rule: resourceRule(
+				[]string{rmv1a1.VerbAll},
+				[]string{rmv1a1.APIGroupAll},
+				[]string{rmv1a1.ResourceAll},
+			)},
+		rmv1a1.PolicyRuleWithSubjects{
+			Subjects: append(
+				users(user.APIServerUser),
+				groups(user.SystemPrivilegedGroup)...,
+			),
+			Rule: nonResourceRule(
+				[]string{rmv1a1.VerbAll},
+				[]string{rmv1a1.NonResourceAll},
+			)},
+	)
+)
+
+// DefaultPriorityLevelConfigurationObjects returns the instances of
+// the ones defined in the KEP, with system-top in position 0 and
+// workload-low in position 1
+func DefaultPriorityLevelConfigurations() []*rmv1a1.PriorityLevelConfiguration {
+	return []*rmv1a1.PriorityLevelConfiguration{
+		DefaultPriorityLevelConfigurationExempt,
+		pl("system-high", rmv1a1.PriorityLevelConfigurationSpec{
+			AssuredConcurrencyShares: 100,
+			Queues:                   128,
+			HandSize:                 6,
+			QueueLengthLimit:         10}),
+		pl("system-low", rmv1a1.PriorityLevelConfigurationSpec{
+			AssuredConcurrencyShares: 30,
+			Queues:                   1,
+			QueueLengthLimit:         1000}),
+		pl("workload-high", rmv1a1.PriorityLevelConfigurationSpec{
+			AssuredConcurrencyShares: 30,
+			Queues:                   128,
+			HandSize:                 6,
+			QueueLengthLimit:         100}),
+		DefaultPriorityLevelConfigurationExempt,
```

Comment:
typo here, "Exempt" instead of "Default"?

Reply:
yes, typo